### PR TITLE
Configmap with Helm

### LINKED ISSUE #1: Adding Config Map
### Willingness to contribute

Yes. I can contribute a fix for this bug independently.

### Feathr version

0.9.0

### System information

- **OS Platform and Distribution (e.g., Linux Ubuntu 20.0)**: macOS Monterey
- **Python version**: python3.9
- **yarn version, if running the dev UI**:

### Describe the problem

Configmap Support

### PR DESCRIPTION
**Pull request by ahlag**
## Description

Resolves https://github.com/feathr-ai/helm-charts/issues/1

## How was this PR tested?

Added configmap for environment variables of Feathr

## Does this PR introduce any user-facing changes?

- [x] No. You can skip the rest of this section.
- [ ] Yes. Make sure to clarify your proposed changes.

### CODE REVIEW

**Comment by ahlag** [comment]
@blrchen 
This is the command I suggested. Let me know if you need any modifications.
https://github.com/feathr-ai/helm-charts/blob/8955f83fa6f6fca9d1f9d612ef910c76e558397c/charts/README.md

Environment Variables based approach
```
$ kubectl exec -it full-coral-feathr-online-55787cd85b-r8lhz -n default -- /bin/sh

/conf # echo $FEATHRCI_HOST
Test
```

**Comment by ahlag** [comment]
@blrchen 
This is the updated test document.
https://github.com/feathr-ai/helm-charts/blob/8e2829c7031e81727fc6c293bb57c5b199c490ce/charts/README.md

File: charts/README.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+# feathr-online
```
Comment (blrchen) [inline charts/README.md]: Please do not move this file. As there will be more charts like feathr-sandbox, feathr-registry under directory charts.

Comment (ahlag) [reply charts/README.md]: Ok sure!

File: charts/feathr-online/templates/configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: feathr-online-config
+  namespace: default
+data:
+  lookup.json: {{ .Files.Get "conf/lookup.json" | toPrettyJson }}
+  pipeline.conf: {{ .Files.Get "conf/pipeline.conf" | indent 4 }}
```
Comment (blrchen) [inline charts/feathr-online/templates/configmap.yaml]: `"conf/lookup.json"` and `"conf/pipeline.conf"` should not be hard coded in template. They should be exposed for passing thru, lookup.json is optional while pipeline.conf is required

File: charts/feathr-online/conf/pipeline.conf [hunk 3]
```diff
@@ -0,0 +1,57 @@
+geoip_demo(ip as string)
```
Comment (blrchen) [inline charts/feathr-online/conf/pipeline.conf]: current lookup.json and pipeline.conf are testing files, should not be packaged into helm chart templates.

**Pull request merged**